docs/DECLARING_DEPENDENCIES.md

```diff
@@ -124,55 +120,6 @@ For an example, see [gax-java#690](https://github.com/googleapis/gax-java/pull/6
 
 See the "Choosing BOMs" section below if you need to import more than one BOM.
 
-## Choosing BOMs
```

Comment:
this section has pretty useful information; can we keep?

Reply:
This is from earlier days before we had a working libraries-bom. On of our OKRs is to make libraries-bom the standard means of consuming these libraries. Developers shouldn't need to think about which BOMs to pick when. Just use libraries-bom is a much simpler and easier to grok story. 

It might be worth adding a section about combining BOMs from multiple projects; e.g. libraries-bom and spring BOM, but that's not what this is about and would be a subject for a separate PR.

Reply:
my thought is that there is a difference btwn best practices for java libraries in terms of how to produce and consume boms, vs. preferred method of consumption for GCP client libraries; The latter is in the c.g.c documentation.

It's still super useful to keep the list of the boms, and also adding libraries bom to the list.  Each individual BOM can be used to override 1 component version as a whole. E.g., in situations where gRPC is causing trouble, simply importing a gRPC BOM would solve the issue.

Reply:
That's a very special case though. It does happen. There's an issue in java-storage right now where people are doing that with the http-java-client, but I'm not sure as written this correctly describes that situation. I want to give a very simple message here:

Use libraries-bom.

That perhaps sometimes  specifies more versions than the users technically needs, but in that case no harm done. It is so much easier for users to consume a simple message like this thna a long, convoluted message of all the possible special cases.

Reply:
I agree with @saturnism .

We should still include the extended message, but we could also "offload" it to a separate page and provide a link to it, so people who do need those details can get to it.